packages/react-storage/src/components/StorageBrowser/components/DropdownMenu.tsx

```diff
@@ -96,6 +96,7 @@ export function DropdownMenu({
                 icon={icon}
                 label={label}
                 onClick={() => {
+                  setIsOpen(false);
```

Comment:
Addresses a regression in the ActionsList composable migration, the menu was not closing on item select in composed StorageBrowser usage